eos-repartition-mbr

```diff
@@ -149,10 +149,12 @@ dd if=/dev/zero of=${root_disk} conv=fsync bs=512 count=1 seek=1
 
 # Remove GPT backup partition table
 dd if=/dev/zero of=${root_disk} conv=fsync bs=512 count=1 seek=$((part_size - 1))
+udevadm settle
 
 # Restore bootloader
 root_mountpoint=$(mktemp -d)
 mount ${root_disk}$((root_partno - 2)) ${root_mountpoint}
 dd if=${root_mountpoint}/boot/grub/i386-pc/boot.img of=${root_disk} bs=446 count=1 conv=fsync
```

Comment:
Also need a settle after this one. The reason is that dd does open() on the disk as read-write and then close(). Upon close(), udev (via inotify) will reprobe the whole disk, which means: first delete all the nodes, second recreate them. Unless you wait for that to complete, the following dd command can occur right at the moment when the nodes have been deleted but not recreated.